webapp/channels/src/components/drafts/panel/panel_body.tsx

```diff
@@ -66,37 +67,43 @@ function PanelBody({
-                <div className='DraftPanelBody__right'>
-                    <div className='post__header'>
-                        <strong>{displayName}</strong>
-                        {priority && (
-                            <PriorityLabels
-                                canRemove={false}
-                                padding='0 0 0 8px'
-                                hasError={false}
-                                persistentNotifications={priority.persistent_notifications}
-                                priority={priority.priority}
-                                requestedAck={priority.requested_ack}
-                            />
-                        )}
+            <div className='DraftPanelBody__post_body'>
+                <ShowMore
+                    text={message}
+                >
+                    <div
+                        onClick={handleClick}
+                        className='post__content'
+                    >
+                        <div className='DraftPanelBody__right'>
+                            <div className='post__header'>
+                                <strong>{displayName}</strong>
+                                {priority && (
+                                    <PriorityLabels
+                                        canRemove={false}
+                                        padding='0 0 0 8px'
+                                        hasError={false}
+                                        persistentNotifications={priority.persistent_notifications}
+                                        priority={priority.priority}
+                                        requestedAck={priority.requested_ack}
+                                    />
+                                )}
+                            </div>
+                            <div className='post__body'>
+                                <Markdown
+                                    options={OPTIONS}
+                                    message={message}
+                                />
+                            </div>
+                        </div>
                     </div>
-                    <div className='post__body'>
-                        <Markdown
-                            options={OPTIONS}
-                            message={message}
-                        />
-                    </div>
-                    {(fileInfos.length > 0 || uploadsInProgress?.length > 0) && (
-                        <FilePreview
-                            fileInfos={fileInfos}
-                            uploadsInProgress={uploadsInProgress}
-                        />
-                    )}
-                </div>
+                </ShowMore>
+                {(fileInfos.length > 0 || uploadsInProgress?.length > 0) && (
+                    <FilePreview
+                        fileInfos={fileInfos}
+                        uploadsInProgress={uploadsInProgress}
+                    />
+                )}
```

Comment:
Just to verify. This no longer lives inside the `DraftPanelBody__right`, and I don't see any relevant css changes about that.

All css that was supposed to be applied to this before is still being applied? (the one that comes to mind more easily is highlights, but not sure if anything else)

Reply:
yes, all css is still valid and looks perfect 👍